src/unitxt/templates.py

```diff
@@ -221,11 +122,21 @@ class YesNoTemplate(Template):
     label_field: str = None
     yes_answer: str = "Yes"
     no_answer: str = "No"
+    yes_answer_references: List[str] = ["Yes", "yes", "1", "True", "true"]
```

Comment:
To be consist for now, let set  

yes_answer_references: List[str]  = ["Yes"]
no_answer_references: List[str]  = ["No"]

and discussion additions in future PRs.

Reply:
changed to match exactly the original behavior